### ISSUE TITLE
Downgrade fails on iPad3,4

### ISSUE THREAD

**Issue by mariobrostech** [LukeZGD/iOS-OTA-Downgrader#85]
I get the following error when trying to downgrade my device. Is there anything I should try to prevent this?
Host computer: iMac15,1 running macOS Mojave 10.14.6
Target device: iPad3,4 running iOS 10.3.3, trying to downgrade to iOS 8.4.1 using OTA blobs

Here's where the errors start happening in the script:
```
* Press POWER or HOME button when screen goes black on the device 
[Log] Finding device in DFU mode... 
root@127.0.0.1's password: 
Connection to 127.0.0.1 closed by remote host.
[Log] Found device in DFU mode. 
./restore.sh: line 345: kill: (2982) - No such process
[Log] Extracting IPSW... 
[Log] Proceeding to idevicerestore... (Enter root password of your PC/Mac when prompted) 
./restore.sh: line 433:  3006 Killed: 9               $idevicerestore -y -e -w $IPSW.ipsw

[Log] Restoring done! 
[Log] Downgrade script done! 
```

The device stays in DFU mode throughout, and the restore never completes.
Thanks in advance for your help!

**Comment by LukeZGD** [comment]
I think this is just the same issue as #62 

Try the `codesign` command again and let me know how it goes

**Comment by mariobrostech** [comment]
Thanks, that seemed to have done the trick!

**Comment by LukeZGD** [comment]
Thanks for letting me know, I will be pushing the change now

**Comment by mariobrostech** [comment]
Hi, I'm re-opening this issue because this problem persists even with the latest version of the script. Here's the log output:

```
[Log] Extracting IPSW... 
[Log] Proceeding to idevicerestore... (Enter root password of your PC/Mac when prompted) 
./restore.sh: line 433: 17522 Killed: 9               $idevicerestore -y -e -w $IPSW.ipsw

[Log] Restoring done! 
[Log] Downgrade script done! 
```

I can confirm that manually codesigning the binary fixes the issue, but the script itself doesn't do this. I also never get prompted for my password when running the script either, which is required for the codesigning).

Thanks again, and sorry for the continued issues :D

**Comment by LukeZGD** [comment]
Fixed in the latest commit 44174c2
